syncthing_gtk/foldereditor.py

```diff
@@ -21,7 +21,7 @@
 	"vversionsPath", "vfsWatcherEnabled", "vcleanoutDays", "vcommand", "vorder",
 	"vminDiskFreePct"
 	]
-VERSIONING_TYPES = set(['simple', 'staggered', 'trashcan', 'external'])
+VERSIONING_TYPES = {'simple', 'staggered', 'trashcan', 'external'}
```

Comment:
Sets can be instantiated directly this way instead of making a list and turning the list into a set.